llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -191,6 +191,12 @@ const AspectsSetTy &getAspectsFromType(const Type *T,
 AspectsSetTy getAspectsUsedByInstruction(const Instruction &I,
                                          TypeToAspectsMapTy &Types) {
   const Type *ReturnType = I.getType();
+  if (auto *AI = dyn_cast<const AllocaInst>(&I)) {
+    // Return type of an alloca is a pointer and in opaque pointers world we
+    // don't know which type it points to. Therefore, explicitly checking the
+    // allocated type insted
```

Comment:
```suggestion
    // allocated type instead
```

Reply:
Fixed in 477a07e42af1254d21d474a1bca224193a90c97f